ci-scripts/JenkinsFile-OAI-Container-GitHub

```diff
@@ -121,7 +121,7 @@ pipeline {
                     branch = 'master'
                     u18_image_tag = 'master'
                     u18_docker_file = 'lte/gateway/docker/mme/Dockerfile.ubuntu18.04'
-                    u18_options = ' --build-arg GIT_PROXY="http://proxy.eurecom.fr:8080"'
+                    u18_options = ''
```

Comment:
GIT_PROXY is still used in the Dockerfiles - should this be cleaned-up too?

Reply:
I would prefer to keep in the dockerfiles. They may be people running in environments where Git Proxy is required.